templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -392,7 +384,10 @@
       "Type": "Task",
       "Resource": "arn:aws:states:::lambda:invoke",
       "Parameters": {
-        "Payload.$": "$",
+        "Payload": {
+          "executionId.$": "$$.Execution.Name",
+          "contentAssets.$": "$.contentAssets"
```

Comment:
Oh, I forgot we did this. We'll need to make changes here as it'll be a bottleneck for the number of assets in a batch. I'll put some tickets in.